google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_datacatalog_tag_factory.py

```diff
@@ -14,15 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+
 from google.cloud import datacatalog
 
 from google.datacatalog_connectors.commons import prepare
+from google.datacatalog_connectors.commons.utils import region_tag_helper
 
-from google.datacatalog_connectors.rdbms.scrape import constants
+from google.datacatalog_connectors.rdbms.common import constants
+from google.datacatalog_connectors.rdbms.prepare.sql_objects import \
+    sql_objects_metadata_config
 
 
 class SQLObjectsDataCatalogTagFactory(prepare.BaseTagFactory):
     __TRUTHS = {1, '1', 't', 'T', 'true', 'True', 'TRUE'}
+    __REGION_TAG_NAME = 'GOOGLE_DATA_CATALOG_METADATA_DEFINITION'
```

Comment:
Should we try to alphabetically sort private attributes whenever possible? Mey vote is yes :). WDYT?